src/algebra/module/linear_map.lean

```diff
@@ -72,6 +72,10 @@ variables [semiring R] [add_comm_monoid M] [add_comm_monoid M₂] [add_comm_mono
 section
 variables [semimodule R M] [semimodule R M₂]
 
+/-- The `distrib_mul_action_hom` underlying a `linear_map`. -/
+def to_distrib_mul_action_hom (f : M →ₗ[R] M₂) : distrib_mul_action_hom R M M₂ :=
+{ map_zero' := zero_smul R (0 : M) ▸ zero_smul R (f.to_fun 0) ▸ f.map_smul' 0 0, ..f }
+
```

Comment:
Could you add a `coe` instance? I don't know the convention for `linear_map`, but it seems the more recently added maps are converging on the `coe` convention over `to_foo_hom`.

Reply:
`linear_map` does not have any other coercions, so I'd rather not muddy the waters by adding one in this PR.